tidb-in-kubernetes/reference/components/tidb-scheduler.md

```diff
@@ -0,0 +1,60 @@
+---
+title: TiDB Scheduler
+category: components
+Summary: Learn how TiDB Scheduler works
```

Comment:
```suggestion
summary: Learn what is TiDB Scheduler and how it works.
category: reference
```
To avoid small mistakes, please double-check the format every time you've done your translation.